ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -66,6 +67,39 @@ Verify User Can Create, Run and Delete A DS Pipeline From DS Project Details Pag
     Delete Pipeline Server    ${PRJ_TITLE}
     Delete Data Science Project    ${PRJ_TITLE}
 
+Verify No Metadata Pods Were Deployed
+    [Documentation]    Because the product includes code that is currently deactivated, it is imperative to prevent any
+    ...         occurrences of disabled elements from appearing unexpectedly.
+    [Tags]    Smoke
```

Comment:
Polarion ID missing

Reply:
No Polarion ID is required. This test will be removed as soon we have the metadata in the UI

Reply:
Uploading the result in Polarion and keeping it on track is enough.

Reply:
In my opinion, you could create the test en Polarion and, once the metadata is enabled in the UI, modify the test (here and in Polarion) to check that the metadata is enabled.  It's not a big deal, so I leave it to your judgement.